tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -83,6 +83,31 @@ var _ = Describe("odo devfile push command tests", func() {
 			helper.CmdShouldPass("odo", "push", "--devfile", "devfile.yaml", "--project", namespace)
 		})
 
+		It("checks that odo push works with a devfile with sourcemapping set", func() {
+			helper.CmdShouldPass("odo", "create", "nodejs", "--project", namespace, cmpName)
```

Comment:
I am not confused why we are only looking for nodejs only? Why not `springboot` ?

Reply:
This is required since `--devfile` isn't supported in `odo create` (see https://github.com/openshift/odo/pull/3118), so this is a necessary step for all tests for devfiles currently.